packages/plugin-rax-pha/src/manifestHelpers.js

```diff
@@ -24,13 +24,35 @@ const retainKeys = [
   'links',
   'scripts',
   'offlineResources',
+  'packageResources',
   'manifestPrefetchExpires',
   'manifestPrefetchMaxAge',
+  'maxAge',
+  'expires',
   'queryParamsPassKeys',
   'queryParamsPassIgnoreKeys',
   'splashViewTimeout',
   'swiperThreshold',
   'requestHeaders',
+  'enablePoplayer',
+  'disableCapture',
+  'enablePullRefresh',
+  'pullRefreshBackgroundColor',
+  'pullRefreshColorScheme',
+  'pullRefresh',
+];
+
+// do not decamelize list
+const doNotDecamelizeKeys = [
```

Comment:
```suggestion
const camelizeKeys = [  // 变量名应该用名词
```

decamelize 已经是 camelize 的反义词了，这里是想表达需要改成小驼峰的 key 嘛？